metals/src/main/scala/scala/meta/internal/bsp/BspResolvedResult.scala

```diff
@@ -14,3 +14,11 @@ case class ResolvedBspOne(details: BspConnectionDetails)
     extends BspResolvedResult
 case class ResolvedMultiple(md5: String, details: List[BspConnectionDetails])
     extends BspResolvedResult
+
+object BspResolvedResult {
+  def fromDetails(details: BspConnectionDetails): BspResolvedResult =
+    details.getName().toLowerCase() match {
+      case "bloop" => ResolvedBloop
```

Comment:
```suggestion
      case BloopServers.name => ResolvedBloop
```
might be better to have a single place where this is defined.

Reply:
Yea good idea. I added this in and replaced the usage of "Bloop" with it in various places.